server/app/views/LoginForm.java

```diff
@@ -75,7 +76,22 @@ public Content render(Http.Request request, Messages messages, Optional<String>
     String title = messages.at(MessageKey.TITLE_LOGIN.getKeyName());
 
     HtmlBundle htmlBundle = this.layout.getBundle().setTitle(title);
-    htmlBundle.addMainContent(mainContent(request, messages));
+
+    DivTag content = div().withClasses(BaseStyles.LOGIN_PAGE);
+
+    content.with(logo());
+
```

Comment:
Can these changes be tested with a browser tests? Even a screenshot tests would be helpful.

Reply:
Just read the description again, so no visual changes. In that case, I think it's alright to check-in.

Reply:
Will do